### PR TITLE
cleanups: rework home page, insert links to external services/references, port files to issues

### LINKED ISSUE #20: Roadmap?
As in autocrypt/autocrypt/issues/324 a Roadmap would be great.

We have core/next-steps.txt, and doc/future_work.rst that should be merged or moved.

### PR DESCRIPTION
**Pull request by hpk42**
- future_work and pgpy_bugs are gone, contents resolved into several issues (along with other issues and the L1-compliance milestone resolves #20)
- rework home page to reflect current aims, add links etc. 
- bump version to 0.8.0.dev1

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -1,58 +1,44 @@
...
+Here are some preliminary underlying aims and goals:
 
-Installation
-------------
+- `Autocrypt Level 1 compliant functionality
+  <https://github.com/hpk42/py-autocrypt/milestone/1>`_ for use by mail user agents (MUAs)
 
-You need the python package installer "pip".  If you
-don't have it you can install it on Debian systems::
+- integrate with re-mailers and other specialized server-side mail agents
```
Comment (azul) [inline doc/index.rst]: what do you mean by remailers? mixnet systems?

Comment (juga0) [reply doc/index.rst]: same question, #34 says "re-mailer integration (#30)", and #30 only talks about mailman

File: doc/index.rst [hunk 2]
```diff
@@ -1,58 +1,41 @@
+Autocrypt support for mail agents
+=================================
 
-Autocrypt Python Reference tool and implementation
-==================================================
+py-autocrypt provides a :doc:`cmdline tool <cmdline>` and a :doc:`Python API <api>`, to help mail agents evolve `Autocrypt <https://autocrypt.org>`_ support.
```
Comment (juga0) [inline doc/index.rst]: Maybe here clarify that intends to implement things that are not autocrpyt goal?

File: doc/index.rst [hunk 3]
```diff
@@ -1,58 +1,41 @@
...
+See :doc:`install` for getting started.
 
-    The py-autocrypt tool is as much in development
-    as the spec itself.  Until we have a 1.0 release
-    everything is subject to change.
+Aims and goals
+--------------
 
-.. toctree::
+- `Autocrypt Level 1 compliant functionality
```
Comment (juga0) [inline doc/index.rst]: So, "Autocrypt Level 1 compliant functionality" is a goal, but doesn't say about the status.

**Pull request merged**